src/lib.rs

```diff
@@ -13,3 +13,11 @@ pub mod metrics;
 #[cfg_attr(iroh_docsrs, doc(cfg(feature = "net")))]
 pub mod net;
 pub mod proto;
+
+#[cfg(feature = "rpc")]
+pub mod rpc;
+#[cfg(feature = "rpc")]
```

Comment:
please add `#[cfg_attr(iroh_docsrs, doc(cfg(feature = "rpc")))]` to these, so it shows up in the docs properly